src/runner/FaabricMain.cpp

```diff
@@ -42,9 +42,6 @@ void FaabricMain::startRunner()
     auto& sch = faabric::scheduler::getScheduler();
     sch.addHostToGlobalSet();
 
-    faabric::util::SystemConfig& conf = faabric::util::getSystemConfig();
-    conf.print();
-
```

Comment:
This was really bloating the tests, so we can now call `print` in specific runners if we feel it necessary (e.g. in the Faasm worker)